algo-builder-tester/assets/algob-tester.py

```diff
@@ -8,6 +8,9 @@
 from algobpy.parse import parse_params
 from pyteal import *  # type: ignore wildcard import
 
+NAMES3 = "btcethxrpadasoldotustltcunitrxfttbchxlmftmetc"
+NAMES4 = "usdtusdclunaavaxbusddogeshibatomnearalgomana"
```

Comment:
Very clever algorithm! Eventually we still need to switch to serial number based, since NAMES4 cannot be more than 128 bytes? But this version should be good for the first version